consensus/istanbul/protocol.go

```diff
@@ -22,13 +22,6 @@ import (
 
 // Constants to match up protocol versions and messages
 const (
-	// No longer supported, can be removed.
-	// The corresponding version upstream (eth/65) is removed in upstream PR #22636.
-	Celo64 = 64 // eth/63 + the istanbul messages
-	Celo65 = 65 // incorporates changes from eth/64 (EIP)
-
-	// Supported versions
-	Celo66 = 66 // incorporates changes from eth/65 (EIP-2464)
 	Celo67 = 67 // incorporates changes from eth/66 (EIP-2481)
```

Comment:
I would also move the comment of the 64 here to keep track of it for the future

```suggestion
	Celo67 = 67 // incorporates changes from eth/66 (EIP-2481) (eth/66 + the istanbul messages)
```

Reply:
makes sense, adding it